tests/comfy_cli/command/generate/test_adapters.py

```
@@ -0,0 +1,320 @@
+    monkeypatch.setattr(upload, "upload_path", fake_upload_path)
+    body = adapters._seedance_build_body(
+        {"prompt": "wave", "image": str(img)},
+        api_key="comfyui-test",
+    )
+    image_part = body["content"][1]
+    assert image_part == {"type": "image_url", "image_url": {"url": "https://cdn/signed-ref.png"}}
+
+
+def test_seedance_build_body_keeps_remote_url_verbatim(monkeypatch):
+    """Remote URLs and data: URIs are pass-through — no re-upload."""
+    from comfy_cli.command.generate import upload
```

Pylint (4.0.5)

[error] Unable to import 'comfy_cli.command.generate'

(E0401)